Update Docs to Explicitly State Subject Attribute Requirement for JIT with Domain Discovery (#5481)
wso2/docs-is: en/includes/guides/organization-management/organization-discovery/email-domain-based-organization-discovery.md

## Map email domains to organizations

> Mapping an email domain to an organization allows for a customized user experience, but this remains optional. When an organization have a mapped email domain, the behavior changes as follows:

Reviewer: ```suggestion
When you map an email domain to an organization, the organization's behavior changes as follows:
```

> - If an organization registers an email domain mapping:

Reviewer: ```suggestion
```

>     <!-- markdownlint-disable MD007 -->

Author: This needs to be disabled, otherwise the sub bullet point are not rendered correctly. It require 4 indentation space, not 2 as the warning suggests.
